packages/eslint-config-skuba/index.js

```diff
@@ -1,151 +1,194 @@
-/** @type {{ js: string[], ts: string[] }} */
+const base = require('eslint-config-seek/base');
 const extensions = require('eslint-config-seek/extensions');
+const jestPlugin = require('eslint-plugin-jest');
+const tsdoc = require('eslint-plugin-tsdoc');
+const eslintPluginYml = require('eslint-plugin-yml');
+const tseslint = require('typescript-eslint');
 
-const { js, ts } = extensions;
-
-/** @type {import('eslint').Linter.Config} */
-module.exports = {
-  extends: ['seek/base'],
-  ignorePatterns: [
-    '**/.eslintrc.js',
-
-    // Gantry resource files support non-standard syntax (Go templating)
-    '**/.gantry/**/*.yaml',
-    '**/.gantry/**/*.yml',
-    '**/gantry*.yaml',
-    '**/gantry*.yml',
-  ],
-  overrides: [
-    {
-      extends: [
-        'plugin:@typescript-eslint/recommended-type-checked',
-        'plugin:@typescript-eslint/stylistic-type-checked',
-      ],
-      files: [`*.{${ts.join(',')}}`],
-      parserOptions: {
-        project: true,
-      },
-      rules: {
-        '@typescript-eslint/consistent-type-exports': 'error',
-        '@typescript-eslint/no-floating-promises': 'error',
-        // allow || on strings and booleans
-        '@typescript-eslint/prefer-nullish-coalescing': [
-          'error',
-          {
-            ignorePrimitives: { string: true, boolean: true },
+const { js: jsExtensions, ts: tsExtensions } = extensions;
+
+module.exports = [
+  { plugins: { jest: jestPlugin } },
+  {
+    ignores: [
+      // Gantry resource files support non-standard syntax (Go templating)
+      '**/.gantry/**/*.yaml',
+      '**/.gantry/**/*.yml',
+      '**/gantry*.yaml',
+      '**/gantry*.yml',
+      '.idea/*',
+      '.vscode/*',
+      '**/.cdk.staging/',
+      '**/.pnpm-store/',
+      '**/.serverless/',
+      '**/cdk.out/',
+      '**/node_modules*/',
+      'coverage*/',
+      'dist*/',
+      'lib*/',
+      'tmp*/',
+    ],
+  },
+  ...base.map(({ plugins: { jest: _jest, ...restPlugins } = {}, ...conf }) => ({
```

Comment:
This could be cleaned up after with an eslint-config-seek change, eslint currently yells if you try to redefine a plugin that's already defined, but if you defined it in a block with `files` you can't use it unless it's exactly the same files 😬